src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <!doctype html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="\images\imac.jpeg"
+        alt="iMac"
+        class="card__image">
+      <div class="card__info">
+        <h2>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      </div>
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+      <div class="testimonials">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+          <div class="stars__item"></div>
+        </div>
+        <div class="testimonials__text">
+          Отзывов: 5
+        </div>
```

Comment:
No need to make a separate block for testimonials, as it most probably won't be reused. Make BEM-blocks for independent logical blocks.